plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events-v2.ts

```diff
@@ -205,6 +205,10 @@ export function addHistoricalEventsExportCapabilityV2(
             createLog(`Starting export ${dateFrom} - ${dateTo}. id=${id}, parallelism=${parallelism}`, {
                 type: PluginLogEntryType.Info,
             })
+            hub.statsd?.increment('historical_export.started', {
+                teamId: pluginConfig.team_id.toString(),
```

Comment:
non-blocking

It is probably fine as there aren't probably many teams running exports, but I guess this _could_ [cause issues with influxdb](https://docs.influxdata.com/influxdb/cloud/write-data/best-practices/resolve-high-cardinality/) if there were lots of teams.

Reply:
Yep - note this is a general comment as most metrics have teamId/plugin associated. It's been fine so far, but high-cardinality metrics have been a pain in the past indeed.